ex2_services/Services/include/services.h

```diff
@@ -94,7 +94,7 @@ union _cnv {
 static char *service_names[] = {"cli_service\0",       "communication_service\0", "time_management_service\0",
                                 "scheduler_service\0", "housekeeping_service\0",  "general_service\0",
                                 "logger_service\0",    "dfgm_service\0",          "adcs_service\0",
-                                "FTP_service\0",       "ns_payload_service\0"};
+                                "FTP_service\0",       "ns_payload_service\0",    "iris_service\0"};

```

Comment:
This will conflict with https://github.com/AlbertaSat/ex2_obc_software/pull/244/files. Sorry

Reply:
Fixed